percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -313,9 +311,7 @@ fix_rules(){
     sed -i 's|CC = gcc-5|CC = /opt/mongodbtoolchain/v4/bin/gcc|' debian/rules
     sed -i 's|CXX = g++-5|CXX = /opt/mongodbtoolchain/v4/bin/g++|' debian/rules
     sed -i 's:release:release --disable-warnings-as-errors :g' debian/rules
-    if [ x"${FIPSMODE}" == x1 ]; then
-        sed -i 's:FIPSMODE=0:FIPSMODE=1:' debian/rules
-    fi
+    sed -i 's:FULL_FEATURED=0:FULL_FEATURED=1:' debian/rules
```

Comment:
If `FULL_FEATURED` is used in neither debian rules file nor in RPM spec file, probably we don't need to set it.